repose-aggregator/docs/src/asciibinder/architecture/system-model.adoc

```diff
@@ -73,101 +61,94 @@ There are several ways to go about getting around this, but one of the most gene
+          hostname="localhost" <!--2-->
+          http-port="8080"/> <!--3-->
+  </nodes>
+
+  <filters/> <!--4-->
+
+  <services/> <!--5-->
+
+  <destinations>
+    <endpoint id="open_repose" <!--6-->
+              protocol="http" <!--7-->
+              hostname="openrepose.org" <!--8-->
+              root-path="/" <!--9-->
+              port="80" <!--10-->
+              default="true"/> <!--11-->
+  </destinations>
 </system-model>
 ----
-<1> Defines a new *Repose* cluster with an ID that must be unique across all defined clusters.
-<2> Defines a new *Repose* node with an ID that must be unique across all defined nodes in the list.
-<3> Defines the hostname of the *Repose* node.
+<1> Defines a new *Repose* node with an ID that must be unique across all defined nodes in the list.
+<2> Defines the hostname of the *Repose* node.
 If set to localhost, any physical node will match and run the *Repose* node.
-<4> Defines the port for the node to listen on.
-<5> Defines a sequence of filters to run.
+<3> Defines the port for the node to listen on.
+<4> Defines a sequence of filters to run.
     In this case, no filters are being used.
-<6> Defines a set of services to run.
+<5> Defines a set of services to run.
     In this case, no services are being used.
-<7> Defines a new endpoint with an ID that must be unique across all defined endpoints in the destinations list.
+<6> Defines a new endpoint with an ID that must be unique across all defined endpoints in the destinations list.
     There must be exactly one default destination endpoint.
-<8> Defines the endpoint's protocol.
+<7> Defines the endpoint's protocol.
     If the protocol is not specified, then internal dispatch is assumed.
-<9> Defines the endpoint's host name.
+<8> Defines the endpoint's host name.
     If not specified, localhost is assumed.
-<10> Defines the endpoint's base path.
-     This will be used in building the URI/path for connecting to the service.
-     Any additional URI info will be appended to this.
-<11> Defines the endpoint's port.
+<9> Defines the endpoint's base path.
+    This will be used in building the URI/path for connecting to the service.
+    Any additional URI info will be appended to this.
+<10> Defines the endpoint's port.
      If not specified (or 0), internal dispatch is assumed.
-<12> Specifies whether or not this is the default destination endpoint.
+<11> Specifies whether or not this is the default destination endpoint.
      The default destination endpoint will be responsible for handling all traffic that is not explicitly routed to a different destination endpoint.
-     Remember that a target can be specified rather than an endpoint if you want to use round-robin routing to a service cluster.
 
 === Filter Activation Determination
 [source,xml]
 .system-model.cfg.xml
 ----
 <system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
-  <repose-cluster id="repose_cluster1"> <!--1-->
     <nodes>
```

Comment:
It looks like you didn't reset the tabbing on this element.

Reply:
Maybe even all of them?

Reply:
Gah, missed this one.

On it.